c++/src/kj/mutex.c++

```diff
@@ -364,6 +376,18 @@ void Mutex::unlock(Exclusivity exclusivity, Waiter* waiterToSkip) {
         }
       }
 
+#ifdef KJ_CONTENTION_WARNING_THRESHOLD
+      uint readerCount;
+      {
+        uint oldState = __atomic_load_n(&futex, __ATOMIC_RELAXED);
+        uint readerCount = oldState & SHARED_COUNT_MASK;
```

Comment:
Need to remove `uint` here.

Reply:
Oof. That would have been bad. Let me add a unit test just to make sure I don't accidentally break it again.